food_basic/locale/tr.txt

```diff
@@ -0,0 +1,40 @@
+# mahmutelmas06@gmail.com
+
+Wheat = Buğday	
+Flour = Un
+Potato = Patates
+Tomato = Domates
+Carrot = Havuç
+Orange = Portakal
+Milk = Süt
+Egg = Yumurta
+Cocoa Bean = Kakao Çekirdeği
+Raw meat = Çiğ et
+Venison = Geyik Eti
+Sugar = Şeker
+Chocolate Powder = Çikolata tozu
+Dark Chocolate = Bitter çikolata
+Milk Chocolate = Sütlü çikolata
+Pasta = Makarna
+Bowl = Kase
+Butter = Yağ
+Cheese = Peynir
+Baked Potato = Fırında Patates
+Pasta Bake = Fırında Makarna
+Raw Pasta Bake = Çiğ Makarna
+chicken Soup = Tavuk çorbası
+tomato Soup = Domates çorbası
+Uncooked tomato Soup = Pişmemiş domates çorbası
+Uncooked chicken Soup = Pişmemiş tavuk çorbası
+apple Juice = Elma suyu
+cactus Juice = Kaktüs suyu
+Rainbow Juice = Atom meyve suyu
```

Comment:
Apparently this means "Atomic Juice". Is this correct?
Google translate says gökkuşağı suyu (although it's not particularly reliable)